packages/Tag/src/Tag.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Close from "@paprika/icon/lib/Times";
+import * as sc from "./Tag.styles";
+import * as types from "./types";
+
+export function Delete({ isDisabled, size, onRemove = () => {} }) {
```

Comment:
Do you need to export this? If so, I think it should be in its own file.  If it's only used here in the Tag component I think it's small enough to leave inline (though maybe it's still best practice to extract it).